azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/implementation/VirtualMachineImpl.java

```diff
@@ -129,47 +127,86 @@ public VirtualMachine create() throws Exception {
         return this;
     }
 
+    @Override
+    public VirtualMachineImpl update() throws Exception {
+        return this;
+    }
+
+    @Override
+    public VirtualMachine apply() throws Exception {
+        return this.create();
+    }
+
+    @Override
+    public void deallocate() throws CloudException, IOException, InterruptedException {
+        this.client.deallocate(this.resourceGroupName(), this.name());
+    }
+
+    @Override
+    public void generalize() throws CloudException, IOException {
+        this.client.generalize(this.resourceGroupName(), this.name());
+    }
+
+    @Override
+    public void powerOff() throws CloudException, IOException, InterruptedException {
+        this.client.powerOff(this.resourceGroupName(), this.name());
+    }
+
+    @Override
+    public void restart() throws CloudException, IOException, InterruptedException {
+        this.client.restart(this.resourceGroupName(), this.name());
+    }
+
+    @Override
+    public void start() throws CloudException, IOException, InterruptedException {
+        this.client.start(this.resourceGroupName(), this.name());
+    }
+
+    @Override
+    public void redeploy() throws CloudException, IOException, InterruptedException {
+        this.client.redeploy(this.resourceGroupName(), this.name());
+    }
+
     /**************************************************.
```

Comment:
yeah, more remnants of the shortcuts prototype, where I didn't have to worry about javadocs or checkstyle :-)
we probably should do one big scrub of everything to clearly distinguish between javadocs and code comments...